README.md

```diff
@@ -4,22 +4,58 @@
 
 This is a Go interface for
 [SPDK](https://github.com/spdk/spdk) which is also a work in progress. Building this requires a
-local SPDK build, so start there first.
+local SPDK (version 18.07) build, so start there first.
+
+To clone the 18.07 SPDK branch:
+
+    git clone --single-branch --branch v18.07.x git@github.com:spdk/spdk.git
 
 ## Current Status
   * Initial support will be for NVMe driver utilities.
 
 ## How to Build
 
 This is a [Go](https://golang.orghttps://golang.org/doc/install)
-project, so a Go development tools are naturally required. We
+project, so the Go development tools are naturally required. We
 recommend the most current Go release available. As of July 2018, the project has been built and tested with Go 1.9.
 
-Setup environment and build. This assumes SPDK shared lib it installed in `/usr/local/lib/libspdk.so`.
+The following steps assume SPDK shared lib is installed in `${SPDK_REPO}/build/lib/libspdk.so`.
 In order to use some of the SPDK API, please also follow [Hugepages and Device Binding](https://github.com/spdk/spdk#hugepages-and-device-binding).
 
-    export GOPATH=$HOME/go
-    export LD_LIBRARY_PATH=/usr/local/lib
-    export PATH=/usr/local/bin:$PATH
-    export CGO_CPPFLAGS="-I/usr/local/include"
-    export CGO_LDFLAGS=-"-L/usr/local/lib -lspdk"
+Clone the repo:
+
+    git clone git@github.com:daos-stack/go-spdk.git
+    export GOSPDK_REPO=${HOME}/go-spdk
+
+Setup and install go-task:
+
+    cd ${GOSPDK_REPO}
+    go get -u -v github.com/go-task/task/cmd/task
+    export PATH=${PATH}:${GOSPDK_REPO}/bin/
+
+Setup environment:
+
+    export GOPATH=${GOSPDK_REPO}
```

Comment:
we shouldn't be changing GOPATH to a specific repo, this should be $HOME/go or it won't work as intended for other Go operations

Reply:
I change the GOPATH back to the default dir.